src/hypofuzz/entrypoint.py

```diff
@@ -71,13 +79,27 @@ def fuzz(
 
     This process will run forever unless stopped with e.g. ctrl-C.
     """
+    if internal_backing_json_file is not None and not dashboard_only:
+        raise click.UsageError(
+            "--internal-backing-json-file must be combined with --dashboard-only"
+        )
```

Comment:
docs say "implies", this is "errors without" - I think I'd be fine with either, but we should also error if you've got a backing json file and pass any other arguments (including unhandled args for forwarding to pytest)